evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -248,12 +254,15 @@ def make_dataframe(
             normalized_values=normalized_values,
             pipeline_features=pipeline_features,
             original_features=original_features,
+            expected_value=expected_value,
         )["explanations"]
 
         # Not including the drill down dict for dataframes
         # 'drill_down' is always included in the dict output so we can delete it
         for d in data:
             del d["drill_down"]
+            if "expected_value" in d.keys():
```

Comment:
nit-pick: As things are implemented now, the `expected_value` is always a key so we can get rid of the check.